src/util/pixelSetter.js

```diff
@@ -1,5 +1,12 @@
+/**  
+ * @param {number} x x-coordinate.
+ * @param {number} y y-coordinate.
+ * @param {object} value array [r, g, b, a]
+ * param {object} pixels NDarray of pixels.
```

Comment:
Oops, you missed a ^`@`^

Reply:
@HarshKhandeparkar Thanks a lot for making my fist PR so easy-going and helping throughout the process :))